src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+import { main } from '../styles/App.module.css';
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
We can totally do away with the use of the `constructor` in the syntax here and we can simply:

```javascript
class App extends React.Component {
  state = {
   total: 0,
   next: null,
   operation: null,
 }
}
```